assets/packages/adminbasics/src/adminbasicsmain.js

```diff
@@ -175,4 +180,4 @@ const AdminCore = function(){
         onLoadRegister();
 }
 
-AdminCore();
+AdminCore();
```

Comment:
Not sure I understand why this change is needed for file manager?

Reply:
we are loading fileManager in the iframe.
the custom styles are loaded first and the iframe is loaded after that.
so styles are not applied.
so, i added file-manager.js in the adminbasicsmain.js and load seagreen css and remix icons style after the iframe is loaded fully.